portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,16 +67,71 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   /*
    * Converts List of comments into a JSON using the gson library.
    */
-  private String convertToJson(List<Comment> comments) {
+  private static String convertToJson(List<Comment> comments) {
     return new Gson().toJson(comments);
   }
 
   /**
    * @return the request parameter, or the default value if the parameter
    *         was not specified by the client
    */
-  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+  private static String getParameter(HttpServletRequest request, String name, String defaultValue) {
     String value = request.getParameter(name);
     return (value == null || value.isEmpty()) ? defaultValue : value;
   }
+
+  /** Returns number of comments entered by the user or -1 if choice is invalid */
+  private static int getNumberOfComments(HttpServletRequest request) {
+    // Get input from the form.
+    String numberOfCommentsString = request.getParameter("number-of-comments");
```

Comment:
use a "private static final String" const for the "number-of-comments"

Reply:
Done.